cli/doc/parser.rs

```diff
@@ -198,6 +200,53 @@ impl DocParser {
     module_decl: &ModuleDecl,
   ) -> Vec<DocNode> {
     match module_decl {
+      ModuleDecl::Import(import_decl) => {
+        let mut imports = vec![];
+        let (js_doc, location) = self.details_for_span(import_decl.span);
+        for specifier in &import_decl.specifiers {
+          use swc_ecmascript::ast::ImportSpecifier::*;
+
+          let import_def = match specifier {
+            Named(named_specifier) => ImportDef {
+              src: import_decl.src.value.to_string(),
+              local: named_specifier.local.sym.to_string(),
+              imported: named_specifier
+                .imported
+                .as_ref()
+                .map(|ident| ident.sym.to_string()),
+            },
+            Default(default_specifier) => ImportDef {
+              src: import_decl.src.value.to_string(),
+              local: default_specifier.local.sym.to_string(),
+              imported: None,
```

Comment:
I think this should be `imported: Some("default".to_string())`. There is no way to tell the difference between this and a namespace import otherwise.

Reply:
Done